stack.yaml

```diff
@@ -40,8 +34,9 @@ extra-package-dbs: []
 #
 # Extra directories used by stack for building
 
-extra-include-dirs:
-- /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/ffi
+## Superfluous?
+# extra-include-dirs:
+# - /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/ffi
```

Comment:
@dorchard git blame says you added this in 2022. Can we remove it?

Reply:
I believe so. Is this redundant for building on recent mac OS X?

Reply:
It builds fine on my 2020 M1 under macOS 14.4.1

Reply:
Looks like we don't need it then.